app/util/bitbucket/upload_attachments.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
```

Comment:
Can we add a check that we are on NFS server, but not on the node.
There is a big chance that the user could start this script on the backend node.